processing/src/main/java/io/druid/query/metadata/metadata/SegmentMetadataQuery.java

```diff
@@ -54,7 +54,8 @@
     SIZE,
     INTERVAL,
     AGGREGATORS,
-    MINMAX;
+    MINMAX,
+    QUERY_GRANULARITY;
```

Comment:
suggest `QUERYGRANULARITY` as this means we can call it `queryGranularity` in the docs.